### DISCUSSION TITLE
How to use suspended domains and Hubs Cloud again

### DISCUSSION THREAD

**Discussion by yukipioa** [mozilla/hubs#5476] [Hubs Cloud Questions]
Hello,

I previously acquired a temporary domain and purchased Hubs Cloud for a test environment.
After the test was over, I suspended or deleted either the domain or the Hubs Cloud (or maybe both, I don't remember) to avoid incurring costs. 

I need to test again and would like to use my old domain and Hubs Cloud again.

I was hoping to use the following records as they are, but do I need to purchase Hubs Cloud and set it up again?

Here is what is left
Domain (valid until December 2022)
Key pair
Stacks

Many Thanks！

**Comment by pattersonbl2** [comment]
You would have to redeploy the cloud formation using the same domains and a backup. 

https://hubs.mozilla.com/docs/hubs-cloud-aws-backup-and-restore.html
